src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/CodePointFilter.cs

```diff
@@ -49,46 +50,46 @@ public CodePointFilter(params UnicodeRange[] allowedRanges)
         }
 
         /// <summary>
-        /// Allows the character specified by <paramref name="c"/> through the filter.
+        /// Allows the character specified by <paramref name="character"/> through the filter.
         /// </summary>
         /// <returns>
         /// The 'this' instance.
         /// </returns>
-        public CodePointFilter AllowChar(char c)
+        public CodePointFilter AllowChar(char character)
         {
-            _allowedCharsBitmap.AllowCharacter(c);
+            _allowedCharsBitmap.AllowCharacter(character);
             return this;
         }
 
         /// <summary>
-        /// Allows all characters specified by <paramref name="chars"/> through the filter.
+        /// Allows all characters specified by <paramref name="characters"/> through the filter.
         /// </summary>
         /// <returns>
         /// The 'this' instance.
         /// </returns>
-        public CodePointFilter AllowChars(params char[] chars)
+        public CodePointFilter AllowChars(params char[] characters)
         {
-            if (chars != null)
+            if (characters != null)
```

Comment:
I suppose it's if you, for example, are chaining this in a big call, and querying other components to get their lists of char's, some of those other components might return null and there's no harm in that.

Or to turn it around, why *disallow* null?